src/Native/Unix/System.Native/pal_errno.c

```diff
@@ -378,6 +379,8 @@ int32_t SystemNative_ConvertErrorPalToPlatform(int32_t error)
             return EHOSTDOWN;
         case Error_ENODATA:
             return ENODATA;
+        case Error_EHOSTNOTFOUND:
+            return -(Error_EHOSTNOTFOUND);
```

Comment:
Just out of curiosity, why is this negative?

Reply:
This is mostly to distinguish from "normal" errno  value. It is used later in SystemNative_StrErrorR(). For now I could probably just just the single value but we may have more down the road.

Reply:
Is that an existing convention for error codes, or is it just something we came up with?

Reply:
it is not convention but something I talk about with @janvorli 
On Windows the hresult codes seems unique cross various APIs.
Something Unixes do not have.